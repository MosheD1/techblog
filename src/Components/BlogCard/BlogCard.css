.defaultBorder {
    border: 1px solid rgba(0, 0, 0, 0.5);
    border-radius: 10px;
    box-shadow: 5px 5px 5px rgba(0, 0, 0, 0.3);
}

.gap {
    margin: 12px;
}

.card {
    transition: all 0.7s linear;
}

.card:hover {
    transform: scale(1.1);
}