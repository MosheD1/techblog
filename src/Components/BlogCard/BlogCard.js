import React from 'react';
import './BlogCard.css'

const BlogCard = ({title}) => {
    return (
        <div className='card defaultBorder gap'>
            <h4>{title}</h4>
        </div>
    );
}

export default BlogCard;