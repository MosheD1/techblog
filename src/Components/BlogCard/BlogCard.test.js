import React from 'react';
import BlogCard from './BlogCard';
import {shallow, render, mount} from 'enzyme';

describe('BlogCard component', () => {
    it('expect BlogCard to render', () => {
        expect(shallow(<BlogCard />)).toMatchSnapshot();
    });
})