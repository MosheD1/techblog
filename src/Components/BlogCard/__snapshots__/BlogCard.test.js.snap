// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogCard component expect BlogCard to render 1`] = `ShallowWrapper {}`;
